requirements.txt

```diff
@@ -0,0 +1,3 @@
+boto3
+discord
+praw
```

Comment:
There's a few more libraries we're going to need. The 2nd version is the latest and first one is the one I have installed. A lot of them come automatically from other libraries so we don't need to specify everything.

Click | 7.0 | 7.1.2
-- | -- | --
PyYAML | 5.1.1 | 5.3.1
aiohttp | 3.6.2 | 3.6.2
aspy.yaml | 1.3.0 | 1.3.0
async-timeout | 3.0.0 | 3.0.1
attrs | 18.2.0 | 19.3.0
beautifulsoup4 | 4.7.1 | 4.9.1
boto3 | 1.7.35 | 1.14.33
botocore | 1.10.35 | 1.17.33
certifi | 2018.11.29 | 2020.6.20
cfgv | 2.0.0 | 3.2.0
chardet | 3.0.4 | 3.0.4
discord.py | 1.3.4 | 1.3.4
docutils | 0.14 | 0.16
filelock | 3.0.12 | 3.0.12
googletrans | 2.4.0 | 3.0.0
identify | 1.4.5 | 1.4.25
idna | 2.6 | 2.10
importlib-metadata | 0.18 | 1.7.0
jmespath | 0.9.3 | 0.10.0
lxml | 4.3.4 | 4.5.2
multidict | 4.7.5 | 4.7.6
nodeenv | 1.3.3 | 1.4.0
packaging | 19.0 | 20.4
pip | 20.0.2 | 20.2
pluggy | 0.12.0 | 0.13.1
praw | 6.0.0 | 7.1.0
prawcore | 1.0.0 | 1.4.0
pre-commit | 1.17.0 | 2.6.0
py | 1.8.0 | 1.9.0
pyparsing | 2.4.0 | 2.4.7
python-dateutil | 2.7.3 | 2.8.1
requests | 2.21.0 | 2.24.0
s3transfer | 0.1.13 | 0.3.3
setuptools | 39.1.0 | 49.2.0
six | 1.11.0 | 1.15.0
soupsieve | 1.9.2 | 2.0.1
toml | 0.10.0 | 0.10.1
tox | 3.13.1 | 3.18.1
translate | 3.5.0 | 3.5.0
translator | 0.0.9 | 0.0.9
update-checker | 0.16 | 0.17
urllib3 | 1.24.1 | 1.25.10
virtualenv | 16.6.1 | 20.0.29
websockets | 6.0 | 8.1
yarl | 1.1.1 | 1.5.1
zipp | 0.5.1 | 3.1.0

Reply:
Looks like everything but `requests` isn't a direct dependency so left those out.